content/src/fieldConfig/check-account-email-page.json

```diff
@@ -0,0 +1,13 @@
+{
+  "checkAccountEmailPage": {
+    "linkAccountText": "Don’t have an account? [Sign up and link your myNewJersey account](https://my.nj.gov/aui/Login)",
```

Comment:
[boulder] The acceptance criteria says:

> And the page will include a "Sign up and link your myNewJersey account" link which will allow users without an existing account to be navigated to Onboarding (https://account.business.nj.gov/onboarding?page=1)

We should not be taking folks to myNJ; they can sign up or link from within our app when they complete onboarding.

Reply:
good catch, that's been updated now

Reply:
We also don't want to implement this as a link in Markdown; we should be using client-side routing to get to that page instead. This improves performance and makes links work on our other domains.